src/app/store/comicses.js

```diff
@@ -0,0 +1,48 @@
+import api from "../../api";
+
+export default {
+	actions: {
+		searchComicses(context, title) {
+			context.commit("setFetching", true);
+			api.searchComicses(title)
+				.then(res => {
+					setTimeout(() => {
+						context.commit("updateComicses", res.data);
+						context.commit("setFetching", false);
+					}, 1000);
+				})
+				.catch(error => {
+					context.commit("setError", error);
+					context.commit("setFetching", false);
```

Comment:
try to use `finally` to reduce code duplication
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally